frontend/src/index.scss

```diff
@@ -128,22 +120,65 @@ body {
 }
 
 .menuItems {
-    padding-top: 150px;
-    .mealtimeTitle {
-        font-weight: bold;
-        font-size: 1.6em;
+    .mealtimeSection {
+        .mealtimeTitle {
+            font-weight: bold;
+            font-size: 1.6em;
+        }
+        .location-title {
+            font-weight: bold;
+            color: #990000;
+            font-size: 1.2em;
+        }
+        .mealtime-icon {
+            width: 50px;
+            &.breakfast {
+                margin-bottom: -10px;
+            }
+            &.lunch {
+                margin-left: 10px;
+                width: 40px;
+            }
+            &.dinner {
+                margin-left: 10px;
+                width: 35px;
+            }
```

Comment:
![image](https://user-images.githubusercontent.com/32281640/182047950-8b48017c-602b-45c5-96e5-ca3fc213d2ee.png)

Brunch is a different size header than the rest of the mealtimes, this is code I wrote though I should have standardized the icon files themselves tbh instead of trying to guess how big they are with CSS

Reply:
Oh yeah we're also using the dinner logo for brunch here. 

Yeah we should just make sure the icons don't have extra whitespace around them.

I'll look into why brunch is larger otherwise but my guess is that its caused by the icon